### PR TITLE
add a demo in README

### PR DESCRIPTION
**Pull request by lfbear**
**What type of PR is this?**

**What this PR does / why we need it**:

Add a demo (a terminal record) in README

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

**Comment by RainbowMango** [comment]
@lfbear Please sign your commit by `git commit -s --amend` and push again.

File: README.md [hunk 1]
```diff
@@ -93,6 +93,12 @@ This guide will cover:
 - Join a member cluster to `karmada` control plane.
 - Propagate an application by `karmada`.
 
+### Demo
+
+There are several demonstrations of common cases.
+
+[![asciicast](https://asciinema.org/a/407612.svg)](https://asciinema.org/a/407612)
```
Comment (RainbowMango) [inline README.md:100]: Does this will jump into asciinema.org?

Comment (lfbear) [reply README.md:100]: Yeah, It does. Let me change another way to embed the demo.

Comment (GitHubxsy) [reply README.md:100]: [karmada  website](https://karmada.io/) maybe it's a place to put the demo.

Comment (lfbear) [reply README.md]: > Yeah, It does. Let me change another way to embed the demo.

Done

Comment (RainbowMango) [reply README.md]: > [karmada website](https://karmada.io/) maybe it's a place to put the demo.

It's ok to save in this repo. No plan shown on the website yet.

**Comment by RainbowMango** [comment]
@kevin-wangzefeng PTAL.
You can get a preview from https://github.com/lfbear/karmada/tree/add_demo.

**Comment by kevin-wangzefeng** [comment]
LGTM

**Pull request merged**